src/components/Chat/PublishModal.tsx

```diff
@@ -153,136 +162,157 @@ export default function PublishModal({
+                  htmlFor="requestVersion"
+                >
+                  {t('Version')}
+                  <span className="ml-1 inline text-accent-primary">*</span>
+                </label>
+                <input
+                  ref={nameInputRef}
+                  name="requestVersion"
+                  className={classNames(inputClassName, {
+                    '!border-error': !isVersionUnique && submitted,
+                  })}
+                  placeholder={t('A version for your request.') || ''}
+                  value={version}
+                  required
+                  type="text"
+                  data-qa="request-version"
+                  onBlur={handleBlur}
+                  onChange={versionOnChangeHandler}
+                />
+                {submitted && (!isVersionUnique || !version.trim()) && (
+                  <div className="text-xxs text-error">
+                    {t(
+                      !isVersionUnique
+                        ? 'Please provide unique version'
+                        : 'Please fill in all required fields',
+                    )}
+                  </div>
                 )}
               </div>
-            )}
+            </section>
+
+            <section className="flex flex-col p-4">
+              <h2>{t('Target Audience Filters')}</h2>
+
+              {[
+                'UserGroup',
+                'JobTitle',
+                'AssignedProjects',
+                'UserGroup',
+                'JobTitle',
+                'AssignedProjects',
+              ].map((v, idx) => (
+                <CollapsableSection
+                  name={v}
+                  dataQa={`filter-${v}`}
+                  key={`filter-${v}-${idx}`}
+                  openByDefault={false}
+                  className="!pl-0"
+                >
+                  TBD
+                </CollapsableSection>
+              ))}
+            </section>
           </div>
-        </section>
-
-        <section className="flex grow flex-col gap-3">
-          <h2>{t('Target Audience Filters')}</h2>
-
-          {[
-            'UserGroup',
-            'JobTitle',
-            'AssignedProjects',
-            'UserGroup',
-            'JobTitle',
-            'AssignedProjects',
-          ].map((v, idx) => (
-            <CollapsableSection
-              name={v}
-              dataQa={`filter-${v}`}
-              key={`filter-${v}-${idx}`}
-              openByDefault={false}
-              className="pl-0"
-            >
-              TBD
-            </CollapsableSection>
-          ))}
-        </section>
-
-        <div className="flex justify-end gap-3">
+          {(type === SharingType.Conversation ||
+            type === SharingType.ConversationFolder) && (
+            <div className="flex w-full flex-col gap-3 p-4 md:max-w-[550px]">
+              <h2>
+                {t(`Files contained in the ${getPrefix(entity).toLowerCase()}`)}
```

Comment:
To use dynamic values in translations in the **i18next.t** we need to do it in this way

```suggestion
                {t(`Files contained in the {{prefix}}`, {prefix: getPrefix(entity).toLowerCase()})}
```
https://www.i18next.com/translation-function/interpolation

Reply:
@denys-kolomiitsev , it works only if we wan't interpolation in result string. Here I generate key and can use javascript interpolation